libraries/AP_NMEA_Output/AP_NMEA_Output.cpp

```diff
@@ -144,13 +150,13 @@ void AP_NMEA_Output::update()
     if ((_message_enable_bitmask.get() & static_cast<int16_t>(Enabled_Messages::GPGGA)) != 0) {
         // format GGA message
         gga_length = nmea_printf_buffer(gga, sizeof(gga),
-                                    "$GPGGA,%s,%s,%s,%01d,%02d,%04.1f,%07.2f,M,0.0,M,,",
+                                    "$GPGGA,%s,%s,%s,%01d,%02d,%04.1f,%07.1f,M,0.0,M,,",
```

Comment:
per https://docs.novatel.com/OEM7/Content/Logs/GPGGA.htm the alt "symbol" shows x.x but the "example" shows 1062.22 so lets keep alt a double-digit.